content/environment_variables.tex

```diff
@@ -30,14 +30,34 @@
     \tabularnewline\hline
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
-    & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Non-negative decimal value (integer or floating point) with an optional
+    character suffix
+    & The value set in this environment variable is interpreted as an
+    implementation-defined size (in bytes) at least as large as the integer
+    ceiling of the product of the numeric prefix and the scaling factor per
+    \ac{PE}. The allowed character suffix for the scaling factor are as follows:
+      \begin{itemize}
+        \item k or K multiplies by \(2^{10}\)  (kibibytes)
+        \item m or M multiplies by \(2^{20}\)  (mebibytes)
+        \item g or G multiplies by \(2^{30}\)  (gibibytes)
+        \item t or T multiplies by \(2^{40}\)  (tebibytes)
+      \end{itemize}
+      For example, string \enquote{20m} is equivalent to the integer value
+      20971520, or 20 megabytes. Similarly the string \enquote{3.1m} is
+      equivalent to the integer value 3250586. Only one multiplier is
+      recognized, so \enquote{20kk} will not produce the same desired result as
+      \enquote{20m}. Usage of string \enquote{.5m} will yield the same result as
```

Comment:
Remove "desired".

Another comment -- this seems to indicate that "20kk" is a valid input.  Do we want to say something like, "Any characters following the multiplier are ignored, so 20kk ..."?  Or say that the multiplier must be the last character in the string?  The current text doesn't indicate whether "20k stupid comments from Jim" is a valid string.  According to the paragraph below, if this is an invalid string, I need to check for it and raise an error.

Reply:
Based on this text specifically the only one multiplier is recognized statement, 20kk or 20K<something something> is still a valid input. It won't error out, it just uses the valid identifiable input characters in the string. 
What do we want? Do we need to consider this as an invalid string?

I prefer the following statement:
Only one multiplier is recognized and any characters following the multiplier are ignored, so ....

To me the multiplier must be the last character statement seems strong and unnecessary.

Reply:
I don't really have a strong preference.  Since we've had this extra characters thing all along and people seem ok with it, your proposed change is probably the easiest option.

Reply:
fixed